apps/editing-toolkit/editing-toolkit-plugin/whats-new/menu/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { ToolsMoreMenuGroup } from '@wordpress/edit-post';
+
+const WhatsNew = () => (
+	<ToolsMoreMenuGroup>
+		<p>What's New copy 2</p>
+	</ToolsMoreMenuGroup>
```

Comment:
Then, update this component to use the fill manually, and give it the right name:

```suggestion
	<Fill name="ToolsMoreMenuGroup">
		<p>What's New copy 2</p>
	</Fill>
```